en/peripherals/frsky_telemetry.md

```diff
@@ -1,27 +1,65 @@
 # FrSky Telemetry
 
-PX4 supports D (old) and S (new) FrSky telemetry. FMUv4 based boards
-(Pixracer) include the required electronics for the interface, older
-boards (Pixhawk) need a UART to S.PORT adapter board.
+FrSky telemetry allows you to access vehicle telemetry and error information on a compatible RC transmitter.
 
+This information includes: flight mode, battery level, RC signal strength, speed, altitude etc. Some transmitters can additionally provide audible and vibration feedback, which is particularly useful for low battery and other failsafe warnings.
 
-## Pixracer
+## Flight Controller Hardware
 
-> **Success** FrSky telemetry auto-starts on Pixracer and
-  detects D or S mode, there is no need to configure anything.
+Pixhawk/PX4 supports D (old) and S (new) FrSky telemetry. FMUv4 based boards (Pixracer) include the required electronics for the interface, older boards (Pixhawk) need a UART to S.PORT adapter board.
 
-For receivers with D-port (D4R-II and similar): Connect the Pixracer
-FrSky TX line to the D4R-II RX line. Also connect GND.
+### Pixracer
 
-For receivers with S-port: Connect the Pixracer FrSky TX and RX lines
-together (solder the wires together) to the X series receiver's S.port
-pin. Also connect GND.
+#### Configuration
 
-## LuaPilot: Telemetry Screen on the Taranis
-If you have a Taranis that can receive the telemetry stream (e.g. a Taranis X9D
-Plus), you can install a script that shows you a nice overview of the received
-telemetry:
-![](../../images/taranis_telemetry.jpg)
+There is no need to configure anything in PX4 itself. Once connected (as discussed below) FrSky telemetry auto-starts on Pixracer and detects D or S mode. 
+
+#### Connecting Receivers with D-port
+
+Connect the Pixracer FrSky TX line to the receiver's RX line. Also connect GND (This applies to D4R-II and similar).
+
+![Radio Connection](../../assets/flight_controller/pixracer/grau_setup_pixracer_radio.jpg)
+
+
+#### Connecting Receivers with S-port
+
+Connect the Pixracer FrSky TX and RX lines together (solder the wires together) to the X series receiver's S.port pin. 
+Also connect GND. The S-port connection is shown below (using the provided I/O Connector).
+
+![Grau b Pixracer FrSkyS.Port Connection](../../assets/flight_controller/pixracer/grau_b_pixracer_frskys.port_connection.jpg)
+
+![Pixracer FrSkyS.Port Connection](../../assets/flight_controller/pixracer/pixracer_FrSkyTelemetry.png)
+
+
+
+### Pixhawk v2

```

Comment:
This applies to any other Pixhawk (i.e. v5 as well)

Reply:
What is "this" are you saying that you always need to set? (ie is it the configuration below - in all cases you need to set SYS_COMPANION to 10) or are you refering to the "Same: GND is not required" comment?

Reply:
But presumably not Pixhawk Pro since that uses TELEM4 and explicitly states that it doesn't need further configuration?
So I think you are saying that normally the S.PORT on Pixhawk is TELEM2?

Reply:
On the Pro, TELEM4 is just the default for frsky telemetry. You can set SYS_COMANION even there (and on Pixracer as well) to run it on TELEM2 instead.

> Does that mean the port is dedicated to S.PORT and always activated for that purpose?

Yes. But this was just because it had no other purpose.

> So I think you are saying that normally the S.PORT on Pixhawk is TELEM2?

Yes